config/www/light-map-panel.js

```diff
@@ -13,15 +13,45 @@ class LightMapPanel extends LitElement {
       panel: { type: Object },
```

Comment:
I think it would be nice to reduce the size of the sensor icons.

At the moment, they cover the info of the  light radial center

Reply:
Fixed in latest commit

Reply:
Nice! The icons look great :rocket: